### PR TITLE
Update execution model to provide data according to specification.

### PR DESCRIPTION
**Pull request by bodommer**

### CODE REVIEW

**Comment by bodommer** [comment]
@yolloz I am fully aware of the fact that it is not a good practice to have verbs in the API endpoint URL, however, I decided to keep it in our case so that the code is cleaner and a single view method only handles a single responsibility without a need for a long if-else to check the method used for the endpoint. If you come up with a good naming, go ahead, but for now, I would keep it this way.

File: backend/crawler/api/tests/get_record_testsuite.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    def test_get_valid_record(self):
+        response = self.client.get('/api/record/get/?record=5')
+        assert 'error' not in response.data
+        assert len(response.data) == 1
+        assert response.data[0]['fields']['label'] == 'my_label'
+        assert len(response.data[0]['fields']['tags']) == 3
+        assert 'b' in response.data[0]['fields']['tags']
+
+    def test_get_invalid_record(self):
```
Comment (yolloz) [inline backend/crawler/api/tests/get_record_testsuite.py:16]: Shouldn't these return 404? It is a valid request but the resource is not found

**Review by yolloz — APPROVED** [review]
Seems OK, the design is not as I would expect it to look like but we have documentation.

**Review by yolloz — APPROVED** [review]
much better, thanks

**Pull request merged**